Dockerfiles/manifests/cluster-agent/cluster-agent.yaml

```diff
@@ -22,5 +22,11 @@ spec:
             value: "true"
           - name: DD_LEADER_ELECTION
             value: "true"
-          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
-            value: <32 characters long token>
+         - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+           value: <Thirty 2 characters long token>
```

Comment:
r/`<Thirty 2 characters long token>`/`<32_CHARACTERS_LONG_TOKEN>`

Reply:
I'll use `<Thirty_2_characters_long_token>` as it is 32 characters long

Reply:
Could you use all caps then to be consistent with the style guide? https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#conventions

many thanks 🙇